### DISCUSSION TITLE
v4.3.2

### DISCUSSION THREAD

**Discussion by Tzahi12345** [Tzahi12345/YoutubeDL-Material#931] [Releases]
Very small update :) Mainly to fix some permissions issues some users were seeing in Docker. Excited to introduce a new Task that will help rebuild your database if it ever breaks. Also for those using armv7, this update reintroduces it.

# New features
* Reintroduced armv7 support
* Added task for rebuilding the database
# Minor additions
* Improved mobile UI for downloads (home screen & downloads manager)
* Translations updates
# Bug fixes
* Fixed issue related to docker UID/GID permissions
* Fixed issue where temp archives would get generated for subscriptions even if none were present

**Full Changelog**: https://github.com/Tzahi12345/YoutubeDL-Material/compare/v4.3.1...v4.3.2

Special thanks to @martadinata666 for helping introduce some of these key fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Tzahi12345/YoutubeDL-Material/releases/tag/v4.3.2'>v4.3.2</a>.</em>

**Comment by FunctionJuan** [comment]
For the Reintroduced armv7 support would that work for Docker containers as well ?

**Reply by Tzahi12345** [reply]
Yup, without Docker it should (and afaik always has) worked just fine with Armv7
